pkg/server/router/credential_test.go

```diff
@@ -274,5 +277,256 @@ func TestCredentialRouter(t *testing.T) {
 		assert.Contains(tt, credStatusMapTwo["statusListCredential"], "v1/credentials/status")
 		assert.NotEmpty(tt, credStatusMapTwo["statusListIndex"])
 
+		// Cred with same <issuer, schema> pair share the same statusListCredential
+		assert.Equal(tt, credStatusMapTwo["statusListCredential"], credStatusMap["statusListCredential"])
+
+		createdSchemaTwo, err := schemaService.CreateSchema(context.Background(), schema.CreateSchemaRequest{Author: "me", Name: "simple schema", Schema: emailSchema})
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, createdSchemaTwo)
+
+		createdCredThree, err := credService.CreateCredential(context.Background(), credential.CreateCredentialRequest{
+			Issuer:     issuer,
+			Subject:    subject,
+			JSONSchema: createdSchemaTwo.ID,
+			Data: map[string]any{
+				"email": "Satoshi2@Nakamoto2.btc",
+			},
+			Expiry:    time.Now().Add(24 * time.Hour).Format(time.RFC3339),
+			Revocable: true,
+		})
+
+		assert.NoError(tt, err)
+		assert.NotEmpty(tt, createdCredThree)
+		assert.NotEmpty(tt, createdCredThree.CredentialJWT)
+
+		credStatusMapThree, ok := createdCredThree.Credential.CredentialStatus.(map[string]any)
+		assert.True(tt, ok)
+
+		assert.Contains(tt, credStatusMapThree["id"], fmt.Sprintf("v1/credentials/%s/status", createdCredThree.ID))
+		assert.Contains(tt, credStatusMapThree["statusListCredential"], "v1/credentials/status")
+		assert.NotEmpty(tt, credStatusMapThree["statusListIndex"])
+
+		// Cred with different <issuer, schema> pair have different statusListCredential
+		assert.NotEqual(tt, credStatusMapThree["statusListCredential"], credStatusMap["statusListCredential"])
+	})
+
+	t.Run("Credential Status List Test No Schemas", func(tt *testing.T) {
```

Comment:
What behavior is being tested here? I took a stab at it. Even trying to do it, I couldn't figure out why there were 3 credentials being created in the test.

See [here](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests) for best practices around naming things. Maybe the test could be split into multiple tests. 

```suggestion
	t.Run("Creating Revocable credentials with different schemas have different status list", func(tt *testing.T) {
```